pkgs/by-name/pr/protoc-gen-go-grpc/package.nix

```diff
@@ -6,22 +6,28 @@
 
 buildGoModule rec {
   pname = "protoc-gen-go-grpc";
-  version = "1.3.0";
+  version = "1.5.1";
```

Comment:
Please separate this in different commits, one for the adoption and another for the bump.